app/services/stack_overflow.rb

```diff
@@ -54,11 +61,13 @@ def is_remote?(title)
   private
 
   def generate_raw_technologies(entry)
-    entry.categories | [term]
+    entry.css('category').map do |category|
+      category.text
+    end
   end
 
   def pull_feed
     url = BASE_URI + term
-    Feedjira::Feed.fetch_and_parse url
+    @rss_feed ||= Nokogiri::HTML(open(url))
   end
```

Comment:
My thought was to memoize it in case we needed to call on it a few different times but we're not doing that, so it's unnecessary.